### DISCUSSION TITLE
Meta Images not being generated?

### DISCUSSION THREAD

**Discussion by iamhectorsosa** [vercel/satori#323] [Q&A]
Hello, everyone. Weird situation here cause I am unsure what might be the issue -> My meta images aren't generated consistently.

I created an endpoint to generate images with the package as shown in the examples. It works, but not consistently. I am unsure whether it is related to Next.js 13 or some catching setting I might have by default on my Vercel deployment.

What do I mean by not consistently? When I share a link that should have a meta image generated with the endpoint, it does work with some platforms, but not with others. It doesn't work with Twitter, Telegram, Open Graph Debugger and Metatags.io. It does work with Facebook, Whatsapp and Slack. It's just strange.

Here is the source code -> [`pages/api/og`](https://github.com/ekqt/ekqt/blob/main/src/pages/api/og.tsx)

Thanks for your input/suggestions/comments!

**Comment by vorn-dev-ni** [comment]
same issue
